bin/less2sass

```diff
@@ -1,22 +1,58 @@
 #!/usr/bin/env ruby
-
+require 'optparse'
 require_relative '../lib/less2sass'
+require_relative '../lib/less2sass/version'
 
-if ARGV.size < 1
-  puts 'Usage: less2sass inputfile [outputfile]'
+options = {}
+
+OptionParser.new do |opts|
+  opts.banner = "Usage: less2sass inputfile.less [options]"
+
+  opts.on("-o", "--output FILE.scss", "name of outputfile") do |o|
+    options[:outputfile] = true
+    options[:outputfilename] = o
+  end
+
+  opts.on("-d", "--delete", "delete inputfile after outputfile is created") do |d|
+    options[:delete] = d
+  end
+
+  opts.on("-v", "--version", "print less2sass version") do |v|
+    options[:version] = v
+  end
+end.parse!
+
+if options[:version]
+  puts "Version: #{Less2Sass::Version}"
+  puts " "
 end
 
-input_file = ARGV[0]
-output_file = ARGV[1]
+if ARGV.size < 1
+  system "less2sass -h"
+else
+  input_file = ARGV[0]
 
-src = File.read input_file
-sass = Less2Sass.convert src
+  src = File.read input_file
+  sass = Less2Sass.convert src
+
+  if options[:outputfile]
+    output_file = options[:outputfilename]
+  else
+    output_file = input_file.gsub("less", "scss")
```

Comment:
We need to make sure this is only replacing the `.less` at the end of the file, since less is an English word that could be part of the filename.